apps/web/src/containers/AnalyticsView/index.tsx

```diff
@@ -0,0 +1,187 @@
+import { useCallback } from 'react';
+import { FormattedMessage } from 'react-intl';
+import {
+  InfiniteScrollDataTable,
+  H1,
+  ScrollToTop,
+} from '@notional-finance/mui';
+import { useAnalyticsTable } from './hooks/use-analytics-table';
+import { Registry } from '@notional-finance/core-entities';
+import { Network, getEtherscanTransactionLink } from '@notional-finance/util';
+import { ReceivedIcon, SentIcon, StackIcon } from '@notional-finance/icons';
+import { formatTokenAmount, formatTokenType } from '@notional-finance/helpers';
+import { useNetworkToggle } from './hooks/use-network-toggle';
+import { useTheme, Box, styled } from '@mui/material';
+import { colors } from '@notional-finance/styles';
+
+export const AnalyticsView = () => {
```

Comment:
Can you rename this file to `NetworkTransactionsView`? I will want to put additional views in here in the future.

Can you rename the parent folder from `AnalyticsView` => `AnalyticsViews`?